### PR TITLE
create WCLI doc

### PR DESCRIPTION
**Pull request by MarnixCroes**
closes https://github.com/zkSNACKs/WasabiDoc/issues/1562

added 715cb0451f1394e89e87d2a6659ab726e63d22ff, updates a link which gives a redirect

### CODE REVIEW

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/.vuepress/config.ts [hunk 1]
```diff
@@ -189,6 +189,7 @@ export default {
                     "/using-wasabi/RPC.md",
                     "/using-wasabi/PayJoin.md",
                     "/using-wasabi/Testnet.md",
+                    "/using-wasabi/WCLI.md",
```
[the remark this answers is not in the document]
Comment (MarnixCroes) [reply docs/.vuepress/config.ts:193]: done

File: docs/using-wasabi/WCLI.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+> Wasabi CLI is currently only available when building from source!
+So it is not available when you use the package.
+
+## Get The Requirements
+
+1. The [same requirements as to build Wasabi from source](/using-wasabi/BuildSource.md#get-the-requirements)
+2. `jq` [command line json processor](https://jqlang.github.io/jq/)
+
+Which can easily be installed on Linux with `sudo apt-get install jq`
+
```
[the remark this answers is not in the document]
Comment (MarnixCroes) [reply docs/using-wasabi/WCLI.md]: removed

File: docs/using-wasabi/WCLI.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+## Usage
+
+To use the wcli, navigate to the _WalletWasabi/Contrib/CLI_ directory.
+In the _CLI_ directory:
+
+```bash
+$ ./wcli.sh command
```
[the remark this answers is not in the document]
Comment (MarnixCroes) [reply docs/using-wasabi/WCLI.md]: you mean it's the first time you try the wcli and it doesn't work for you on win?
it's work on Linux.

**Comment by lontivero** [comment]
Wait wait no. The `wcli` is a script for devs only and the idea is to make things a little bit easier for us to interact with the json rpc server. This is not something for others to know about, it is so so low quality, so buggy and so shitty that it could harm our reputation if people start using it. Also, given they will find hundred million issues they will report those problems for us to solve them when we really don't want to do anything with that script. Please don't document it, keep it unknown and unpublished.

File: docs/using-wasabi/WCLI.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/WCLI.md]: Yes it doesn't work for me on win.

**Pull request closed without merging**